### PR TITLE
Instant values are now converted as days-since-epoch.

### PR DESCRIPTION
**Pull request by vaskalan**
This applies to new variables that are to be imported into M2.0 after production.
Date values for migrated datasets from M1.5 are still converted as strings.

### CODE REVIEW

File: dataset_builder/run_dataset_conversion_only.py [hunk 1]
```diff
@@ -21,6 +21,12 @@ def main(argv):
     log_config.setup_logger_for_import_pipeline(config)
     initial_logging(config, data_file, temporality, data_type)
 
+    # This should be removed after migration of datasets to M2.0 is finalized.
```
Comment (DanielElisenberg) [inline dataset_builder/run_dataset_conversion_only.py:24]: :100:

File: dataset_builder/steps/dataset_converter.py [hunk 2]
```diff
@@ -84,17 +84,23 @@ def __create_table_for_partitioned_parquet(data_path: str, data_type: str) -> pa
...
     stop = epoch + timedelta(days=temporal_coverage["stop"])
     logger.info(f"Generating epoch dict from {start} to {stop}")
-    epoch = datetime.utcfromtimestamp(0)
     while start <= stop:
         days_since_epoch_for[start.strftime("%Y%m%d")] = str((start - epoch).days)
         start += timedelta(days=1)
     return days_since_epoch_for
 
 
+def __generate_epoch_day(date: str, date_do_not_exist: list) -> str:
```
Comment (DanielElisenberg) [inline dataset_builder/steps/dataset_converter.py:96]: Can we change from date_do_not_exist => empty_values in this func as well?

**Review by DanielElisenberg — APPROVED** [review]
Looks good! 👍🏻

**Review by pawbu — APPROVED** [review]
Looks good

**Pull request merged**